01-factoring-overview.ipynb

```diff
@@ -188,7 +188,7 @@
    "outputs": [],
    "source": [
     "and_bqm = dbc.stitch(and_csp)\n",
-    "and_bqm.remove_offset()\n",
+    "and_bqm.offset = 0\n",
```

Comment:
What's the reason for this change?

Reply:
`.remove_offset` is removed in dimod 0.10

Reply:
I guessed that might be the case but when I look at https://github.com/dwavesystems/dimod/blob/b8a91bd810d632ace95efa90fba7993d918a8a45/dimod/core/bqm.py#L462 I didn't see a deprecation. Is that somewhere else?

Reply:
@arcondello might know.

Reply:
No, 0.9 -> 0.10 is a backwards breaking change afterall and this is one of those changes. If it's a big problem we can restore it with a deprecation warning but we'll still need to change it here to avoid the warning.

Reply:
I suspect this function might have been used similarly in other code. Should we maybe have a deprecation warning now and take it out in the next release?

Reply:
Sure, though we still need to make this change here and everywhere else to avoid raising the warning.